### DISCUSSION TITLE
An easy way to export/copy of a portion of tree (mutiple tabs, group tabs) in the same panel as markdown?

### DISCUSSION THREAD

**Discussion by directentis1** [mbnuqw/sidebery#1761] [Ideas]
Currently, using "Copy URLs" in context menu will copy URLs, regardless of whether they are nested in a tree structure, to flat format.

Is it possible to keep that structure intact when copying them or copying them as markdown format links (including URL and titles)?

Perhaps a more general problem: **_can SideBery support its own API and customize operations with tabs, menus... by user scripting?_**

I want to handle use cases, typically where I often open links in a tree when researching, and copy them all to refer back to, or to add as citations in my notes or papers.

Another case is to send those links to external programs, through other add-ons such as Zotero, Anki-connect to conveniently manage references, citations or create flashcards with the content provided that has sent from Sidebery - via context menu or customized keyboard shortcuts and dialogs? 🤔

**Comment by albino1** [comment]
Maybe in the settings try going to Snapshots and turning Full Tree Structure in Markdown on and then just get it from the saved snapshot?

**Reply by directentis1** [reply]
Yeah, it's the only way... But you know, I just need each group of related tabs instead of having to drag hundreds of tabs just to find them 🤷

**Comment by edrex** [comment]
I want this. Markdown subtree copy, but also org-mode format. Other ppl might want mediawiki syntax too.

**Comment by ForDefault** [comment]
I would just look at [foxytab](https://addons.mozilla.org/en-US/firefox/addon/foxytab/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search)
Especially the **custom copy urls** setting.

**Reply by insipidlight** [reply]
this doesn't work if you have userChrome mods that hide the tabbar though. it would be nice to shift/ctrl-select individual tabs in Sidebery instead

**Comment by directentis1** [comment]
Solved in newest release.
https://github.com/mbnuqw/sidebery/releases/tag/v5.4.0
